File: Lab2/src/KI/Kulyk/Lab2/Display.java
package KI.Kulyk.Lab2;

/**
 * Клас для опису дисплея.
 */
public class Display {
    private String type;
    private double size;

    public Display(String type, double size) {
        this.type = type;
        this.size = size;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public double getSize() {
        return size;
    }
}

File: Display.py
class Display:
    def __init__(self, display_type, size):
        self.type = display_type
        self.size = size

    def get_type(self):
        return self.type

    def set_type(self, display_type):
        self.type = display_type

    def get_size(self):
        return self.size